.github/workflows/build.yml

```diff
@@ -25,23 +25,9 @@ jobs:
     steps:
     - name: Check out repository
       uses: actions/checkout@v4
-    - name: Install action dependencies
-      run: |
-        apt-get update -qq
-        apt-get install -qq sudo wget
-    - name: Setup caches
-      uses: actions/cache@v4
-      id: cache-id
+    - name: Set up OpenFOAM
+      uses: gerlero/setup-openfoam@v1
       with:
-        path: ${{ runner.temp }}/cache-directory
-        key: ${{ runner.os }}-cache-dependencies-v1.0
-    - name: Install OpenFOAM (considering cache)
-      uses: airvzxf/cache-anything-new-action@v1.0.1
```

Comment:
It does attempt to cache the deb via the [apt-deb-cache](https://github.com/marketplace/actions/cache-and-install-apt-deb-packages) action, but currently this doesn't work when running inside a container (https://github.com/gerlero/setup-openfoam/pull/13). My goal is to fix this in the near future though.

Reply:
Alright, happy to not have to maintain the caching mechanism here anyway! :smile: 
If you add more details in the issue, I am happy to help debug it.

This is not a blocking issue, in my opinion. I think the benefit is much higher.

Reply:
Not to miss it, the cache-anything-new-action should work but the requirement that you pass a script (instead of simple commands) makes it more difficult to configure. Not that it wouldn't be possible to do so, but that's what pushed me towards trying to roll my own.

Reply:
For me, the cache-anything-new-action was just an easy way to say "I don't know what exactly I need to cache, so this automated method should work".

Reply:
@MakisH caching is now working even in containers (I ended up rolling my own @ [gerlero/apt-install](https://github.com/gerlero/apt-install)).